docs/README.md

```diff
@@ -53,6 +53,16 @@ Use metadata to show or hide input/output cells in notebooks.
 -   `remove_all_output`, e.g. for the cells outputting information after calling `.save()` or `.upload()`.
 -   `remove_cell`, e.g. when you want something executed without displaying input or output.
 
+## Contributing
+
+The docs include many Jupyter notebook (`ipynb`) files that must have their output and metadata (except `tags`) cleared before a commit.
+
+The `.gitattributes` in the `/docs` directory expects the `clear-notebook-output` filter. Please add it to your config with the following command:
+
+```bash
+git config filter.clear-notebook-output.clean "poetry run jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --ClearMetadataPreprocessor.preserve_cell_metadata_mask tags --to=notebook --stdin --stdout --log-level=ERROR"
```

Comment:
You'll need to `cd docs/; poetry run ...`, as it looks like filters run from the workspace directory